x/farming/keeper/hooks_test.go

```diff
@@ -0,0 +1,43 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmosquad-labs/squad/v2/x/farming/types"
+
+	_ "github.com/stretchr/testify/suite"
+)
+
+var _ types.FarmingHooks = &MockFarmingHooksReceiver{}
```

Comment:
Minor suggestion since this is a test code:
```suggestion
var _ types.FarmingHooks = (*MockFarmingHooksReceiver)(nil)
```

Ref. https://go.dev/doc/faq#guarantee_satisfies_interface

Reply:
Thanks! I will apply the suggestion.